docs/configuration/prometheus-frontend.yml

```diff
@@ -56,3 +56,5 @@ frontend:
 
   # The Prometheus URL to which the query-frontend should connect to.
   downstream_url: http://prometheus.mydomain.com
+  # Max body size for downstream prometheus
+  downstream_max_body_size: 10485760
```

Comment:
I would remove this. Let's keep our examples as simples as possible.

Reply:
Done